src/services/api/media.js

```diff
@@ -18,31 +14,18 @@ const media = {
     const nextPage = response?.next
     return { ...response, nextPage, lastPage }
   },
+  getAudio: async ({ sitename, id }) =>
```

Comment:
Moved these 3 methods after the `get` method. no modification here just re-ordering.
Added an upload endpoint towards the end of the file.